apps/marketplace/common-util/List/RegisterForm/index.jsx

```diff
@@ -116,7 +116,7 @@ const RegisterForm = ({ isLoading, listType, handleSubmit, handleCancel }) => {
           validateFirst
           rules={[
             {
-              required: false,
+              required: true,
```

Comment:
do we use it as a shared form for minting a component and a blueprint, and for blueprint it's required?